### DISCUSSION TITLE
10, 20, 30ms VAD model

### DISCUSSION THREAD

**Discussion by siriusht** [snakers4/silero-vad#363] [Q&A] [answered]
## ❓ Questions and Help
The VAD model used in your example is based on 64ms. 
there are two questions:
1. Can I input 10ms or 20ms or 30ms audio data into this model?
2. Could you please share 30ms model?
3. Does it support speech language Chinese?

**Comment by snakers4** [comment, marked as answer]
Hi,

> Can I input 10ms or 20ms or 30ms audio data into this model?
> Could you please share 30ms model?

The preferred chunk sizes are described here:
https://github.com/snakers4/silero-vad/blob/563106ef8cfac329c8be5f9c5051cd365195aff9/utils_vad.py#L205-L208

> Does it support speech language Chinese?

It should be language agnostic, and I believe Chinese was in the train dataset, albeit a small amount of data.

**Reply by siriusht** [reply]
I got it, thanks.
